src/Styleguide/Elements/__tests__/Select.test.tsx

```diff
@@ -11,16 +11,16 @@ describe("Select", () => {
 
   it("is responsive", () => {
     const small = mount(
-      <Boot initialMatchingMediaQueries={["xs"]}>
+      <MockBoot breakpoint={"xs"}>
```

Comment:
#superminor - For consistency with the above changes, maybe just `breakpoint='xs`?